### PR TITLE
Include 4 new properties on atom images

### PR DESCRIPTION
**Pull request by ajwl**
We want to pick up and use some extra properties of Images on the Grid.
Images should now have these properties, if available in Grid metadata: 

- copyright
- source
- photographer
- suppliersReference

[Trello ticket](https://trello.com/c/GexFvYIf/240-add-new-fields-for-atom-images-adding-fields-for-atom-images-copyright-source-photographer-suppliersreference-so-rcs-can-better)

### CODE REVIEW

File: public/js/util/imageHelpers.js [hunk 1]
```diff
@@ -11,6 +11,21 @@ export function parseMimeType(mimeType) {
...
+  const source = metadata.source;
+  const photographer = metadata.photographer;
+  const suppliersReference = metadata.suppliersReference;
+  return {
+    credit,
+    copyright,
+    source,
+    photographer,
+    suppliersReference
+  }
```
Comment (tomrf1) [inline public/js/util/imageHelpers.js]: the linter wants a semicolon here

File: public/js/util/imageHelpers.js [hunk 2]
```diff
@@ -35,16 +50,20 @@ function parseAsset(asset, credit) {
       width: asset.dimensions.width,
       height: asset.dimensions.height
     },
-    credit: credit
+    credit: metadata.credit,
```
Comment (shaundillon) [inline public/js/util/imageHelpers.js]: You could probably use destructuring here to make it a bit nicer. e.g:

`const { credit, copyright, source, ... }  = metadata`

**Comment by ajwl** [comment]
How do we want the photographer field to show?

Either: `Tim Askew`
Or, like the byline field in Grid? `Tim Askew/National Trust Images`

File: public/js/util/imageHelpers.js (hunk 2 above)
Comment (ajwl) [reply public/js/util/imageHelpers.js]: sure, will add now

**Comment by ajwl** [comment]
[Add new fields for atom images (Adding fields for atom images (`copyright`, `source`, `photographer`, `suppliersReference`) so RCS can better process rights](https://trello.com/c/GexFvYIf/240-add-new-fields-for-atom-images-adding-fields-for-atom-images-copyright-source-photographer-suppliersreference-so-rcs-can-better)

**Comment by ajwl** [comment]
Have merged related changes to the Thrift Model on:
Content-Atom: https://github.com/guardian/content-atom/pull/121
Atom-maker: https://github.com/guardian/atom-maker/pull/47

**Comment by tomrf1** [comment]
👍

**Pull request merged**